lib/Core/Resources/translations/messages.en.xlf

```diff
@@ -24,26 +24,59 @@
             </trans-unit>
             <trans-unit id="6">
                 <source>Range</source>
-                <target>Range</target>
+                <target>range</target>
             </trans-unit>
             <trans-unit id="7">
                 <source>ExcludedValue</source>
-                <target>ExcludedValue</target>
+                <target>excluded-value</target>
             </trans-unit>
             <trans-unit id="8">
                 <source>ExcludedRange</source>
-                <target>ExcludedRange</target>
+                <target>excluded-range</target>
             </trans-unit>
             <trans-unit id="9">
-                <source>Comparison</source>
-                <target>Comparison</target>
+                <source>Compare</source>
+                <target>comparison</target>
             </trans-unit>
             <trans-unit id="10">
                 <source>PatternMatch</source>
-                <target>PatternMatcher</target>
+                <target>pattern-matcher</target>
+            </trans-unit>
+            <trans-unit id="11">
+                <source>No default field was configured.</source>
+                <target>No default field was configured.</target>
+            </trans-unit>
+            <trans-unit id="12">
+                <source>Field {{ field }} is not registered in the FieldSet or available as alias.</source>
+                <target>Field {{ field }} is not registered in the FieldSet or available as alias.</target>
```

Comment:
This is one is already present in `validators.[en].xlf`. Not sure what do to here.

Reply:
Move all translations (with the exception of explicit validator messages 'This value is not a valid birthday.') to RollerworksSearch translation domain?

Reply:
I moved all violation translations to the validator domain, all the Search specific translations are now placed in the RollerworksSearch domain.

Note that syntax errors and formatting errors, that would always be false are placed in the later. Constraints for maximum number of values, groups, nesting depth, and accept value-type are placed in validators.